static/src/js/providers/AppContextProvider/AppContextProvider.jsx

```diff
@@ -32,35 +37,97 @@ const AppContextProvider = ({ children }) => {
   const [draft, setDraft] = useState()
   const [savedDraft, setSavedDraft] = useState()
   const [user, setUser] = useState({})
+  const [providerIds, setProviderIds] = useState([]) // State to hold providerIds
 
   const { keywords } = keywordsContext
 
+  const contextValue = useContext(NotificationsContext)
+  const { addNotification } = contextValue || {}
+
   useEffect(() => {
     setUser({
       name: 'User Name',
       token: 'ABC-1',
-      providerId: 'MMT_2'
+      providerId: 'MMT_1'
```

Comment:
I am not sure we want to set a default provider here any more. I think we want to leave the provider undefined until the provider query responds, then set the provider to whatever the first available provider is.